packages/cli/src/services/public-api-key.service.ts

```diff
@@ -107,14 +108,17 @@ export class PublicApiKeyService {
 
 			if (!user) return false;
 
-			try {
-				this.jwtService.verify(providedApiKey, {
-					issuer: API_KEY_ISSUER,
-					audience: API_KEY_AUDIENCE,
-				});
-			} catch (e) {
-				if (e instanceof TokenExpiredError) return false;
-				throw e;
+			// Legacy API keys are not JWTs and do not need to be verified.
+			if (isJwt(providedApiKey)) {
```

Comment:
this is not really checking for JWTs, but rather for UUIDs, no?

Reply:
yep. maybe we should check for `providedApiKey.startsWith('eyJ')` instead.

Reply:
also, we could move this entire block above, and use the `sub` as `userId` for looking up the user, instead of doing the join in `getUserForApiKey` for these new keys.

Reply:
> also, we could move this entire block above, and use the sub as userId for looking up the user, instead of doing the join in getUserForApiKey for these new keys.

Will address this in another PR.